packages/hdwallet-vultisig/src/adapter.ts

```diff
@@ -0,0 +1,85 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+
+import { VultisigBftProvider, VultisigEvmProvider, VultisigSolanaProvider, VultisigUtxoProvider } from "./types";
+import { VultisigHDWallet } from "./vultisig";
+
+declare global {
+  interface Window {
+    vultisig?: {
+      ethereum?: VultisigEvmProvider;
+      bitcoin?: VultisigUtxoProvider;
+      litecoin?: VultisigUtxoProvider;
+      dogecoin?: VultisigUtxoProvider;
+      bitcoincash?: VultisigUtxoProvider;
+      zcash?: VultisigUtxoProvider;
+      dash?: VultisigUtxoProvider;
+      solana?: VultisigSolanaProvider;
+      keplr?: VultisigBftProvider;
+    };
+  }
+}
```

Comment:
q: can we do without a global window namespace pollution here?

Not the biggest deal as it's just types, but wondering if that could override other declarations and rug things like e.g `window.ethereum` types in other places.